workspace/src/github.com/darkcrux/go-ceph-rest-api/handler_pool.go

```diff
@@ -0,0 +1,71 @@
+package main
+
+import (
+	"fmt"
+
+	"net/http"
+
+	"encoding/json"
+	"github.com/gorilla/mux"
+)
+
+const (
+	// actions
+	create = "create"
+	// get = "get"
+	// delete = "delete"
+
+	// methods
+	// get = "GET"
+	// post = "POST"
+	put = "PUT"
+	// delete = "DELETE"
+)
```

Comment:
maybe we can use a better naming convention for this:

ActionCreate, ActionGet
MethodPut, MethodPost

or CreateAction, GetAction.

hmmm.. naming is hard. :D